core/platform/models.py

```diff
@@ -392,8 +392,12 @@ def import_secrets_services(cls) -> ModuleType:
         Returns:
             module. The cloud_secrets_services module.
         """
-        from core.platform.secrets import cloud_secrets_services
-        return cloud_secrets_services
+        if constants.EMULATOR_MODE:
+            from core.platform.secrets import cloud_secrets_services
```

Comment:
Will cloud secrets ever work in a local environment? Also, is this logic correct--shouldn't we be importing cloud secrets by default rather than dev mode? I'm assuming production has both DEV_MODE and EMULATOR_MODE disabled.

Reply:
This has been changed to gate on dev_mode, instead.